### ISSUE TITLE
Mint 21: when using inbuilt dark themes, windows can be hard to distinguish one from another

### ISSUE THREAD

**Issue by LinuxOnTheDesktop** [linuxmint/mint-themes#402]
A screenshot, using (as the image itself shows) the Mint-Y dark theme:

![image](https://user-images.githubusercontent.com/47056543/186961772-3b91807b-67c0-49c4-967f-93c6460e4486.png)

Another screenshot, using the Mint-Y-Legacy dark theme:

![image](https://user-images.githubusercontent.com/47056543/186961973-10405132-c794-491e-8d00-b26d88e09dd7.png)

Note in both screenshots the way that the top corners of the topmost window is hard to distinguish from that which lies behind them. Please note also that, when one is looking at a whole screen, it is harder - harder than in the partial screenshots - to make the distinction.

I presume that the solution is in one way or another to darken the outline of the topmost window. (I feel also that the window decorations are slightly too small and - perhaps - set slightly too far from the windows top border, but those are gripes for another occasion, I suppose.)

Cf.: #214; #253.

**Comment by LinuxOnTheDesktop** [comment]
Except, ah, _sometimes_ the contrast is alright:

![image](https://user-images.githubusercontent.com/47056543/186963716-0ffaf577-b395-44ee-bcd5-c91c09602e37.png)

(except that, evidently enough, the anti-aliasing on the top-right and top-left corners is crude and ugly).

**Comment by SebastJava** [comment]
I agree. When looking at the CSS or SASS files, I noticed the shadows for dark variants are not as strong as for the light variants.

The Mint design team is very obsessed with this idea of making things "not too distracting", "not too colorful", and "not too contrasted". This dark theme is already just made of dark grays on dark grays everywhere, without any pure black. There is hardly any contrasts in there. And, on top of that, the shadows are "not so strong".

Yes, I partly understand this need to avoid pure black backgrounds, as explained in https://uxmovement.com/content/why-you-should-never-use-pure-black-for-text-or-backgrounds/ But all colors and all contrasts can only be appreciated when there are some comparisons, some references available. I mean, the text could be pure white on a dark gray background like `#404040`. But then the window borders could be much darker, and the shadows could go as far as... absolute black, i.e. `#000000` ! Why not?

**Comment by SebastJava** [comment]
> Except, ah, sometimes the contrast is alright: (image)
(except that, evidently enough, the anti-aliasing on the top-right and top-left corners is crude and ugly).

Maybe that's just another occurrence of one of my big issues here again: https://github.com/linuxmint/mint21-beta/issues/120 Of course, those shadow issues are not much visible on Mint-Y-Dark, since all contrasts are sooo subtle in there...

P.S.: I am still waiting for these https://github.com/linuxmint/mint21-beta/issues to be fixed :(

**Comment by LinuxOnTheDesktop** [comment]
The relations between windows are much clearer on Windows 10. Screenshot:

![image](https://user-images.githubusercontent.com/47056543/187030376-4190704a-9341-4810-bca8-0a6503cc0637.png)

We should be able to learn from Windows on this front.
